contracts/test/gateway/MockEIP20Gateway.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.0;
+
+import "../../gateway/EIP20Gateway.sol";
+import "../../lib/IsMemberInterface.sol";
+
+contract MockEIP20Gateway is EIP20Gateway {
```

Comment:
I think after removing gateway linking this is not needed anymore.

Reply:
Wow good catch 🐙